archiver/src/main/java/com/netflix/conductor/archiver/job/QueueMessageJob.java

```diff
@@ -0,0 +1,47 @@
+package com.netflix.conductor.archiver.job;
+
+import com.netflix.conductor.archiver.config.AppConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static org.apache.commons.collections.CollectionUtils.isNotEmpty;
+
+public class QueueMessageJob extends AbstractJob {
+    private static final Logger logger = LogManager.getLogger(QueueMessageJob.class);
+    private static final String QUERY = "select id from queue_message qm\n" +
+            "where qm.queue_name in ('_deciderqueue', '_sweeperqueue')\n" +
+            "and qm.message_id in (select wf.workflow_id from workflow wf\n" +
+            "where wf.workflow_type in ('deluxe.dependencygraph.sourcewait.process.1.0', 'deluxe.dependencygraph.sourcewait.process.1.1',\n" +
```

Comment:
Read workflows from the vault variable (e.g. `archiver_cleanup_message_workflows`) similar how `aurora_metrics_main_workflows` is used. Update `archiver/src/main/java/com/netflix/conductor/archiver/config/AppConfig.java` and `docker/archiver/bin/startup.sh` to reflect that variable. Use default (those 4 workflows) if not defined